File: ai/src/main/java/org/pingel/bayes/Value.java
package org.pingel.bayes;

public class Value
implements Comparable<Value>
{

	private String v = null;
	
	public Value(String v)
	{
		this.v = v;
	}
	
	public int compareTo(Value other)
	{
		return v.compareTo(other.v);
	}
	
	public String toString()
	{
		return v;
	}
}

File: gestalt/src/main/scala/org/pingel/gestalt/core/Value.scala
package org.pingel.gestalt.core

class Value(v: String)
extends SimpleForm(new Name(v)) // may not be what was intended in original
// with Comparable[Value] 
{

  def compareTo(other: Value): Int = v.compareTo(other.getV)

  def getV(): String = v

  override def toString() = v

}

